### ISSUE TITLE
add all-contributors bot

### ISSUE THREAD

**Issue by cassgvp** [cassgvp/WIN-Open-Neuroimaging-Community#24]
We need a quick method of adding contributors without going through a full pull request. All-contributors would work for this?

**Comment by Ruth-ikegah** [comment]
Yeah, it will work. Although hen mentioned, all contributors automatically create the PR which you will still have to merge.
Then I think it's a maintainer that has the right to install it on a repository.

Kinkly check this:  https://allcontributors.org/docs/en/bot/installation
